semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -238,7 +251,35 @@ def __init__(self, group):
 class MetricSemanticConvention(BaseSemanticConvention):
     GROUP_TYPE_NAME = "metric"
 
-    allowed_keys = ()
+    allowed_keys: Tuple[str, ...] = BaseSemanticConvention.allowed_keys + ("metrics",)
+
+    class Metric:
+        def __init__(self, metric, parent_prefix):
+            self.id: str = metric.get("id")
+            self.fqn = "{}.{}".format(parent_prefix, self.id)
+            self.instrument: InstrumentKind = InstrumentKind[metric.get("instrument")]
+            self.units: str = metric.get("units")
+            self.brief: str = metric.get("brief")
+
+            if None in [metric.get("instrument"), self.id, self.units, self.brief]:
+                raise ValueError
```

Comment:
Missing exception message.
Consider using `raise ValidationError.from_yaml_pos` like in other locations.